Runtime/Core/Scripts/Extensions/EnumExtensions.cs

```diff
@@ -24,5 +25,15 @@ public static class EnumExtensions
 
             return default;
         }
+
+        public static string GetDescription(this Enum value)
```

Comment:
we actually already have this function in the BodyShapeSelectionElement.cs class. However I think that a better location for it is here as you have it. So can you remove the other one from BodyShapeSelection and make it use this one? 
<img width="819" alt="Screenshot 2024-06-18 at 8 30 58 am" src="https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/96b43f07-6c59-4bd7-bbe9-41ed2c27394c">